README.md

```diff
@@ -16,6 +16,15 @@ Tested with a Kubernetes v1.10 Minikube setup:
 $ minikube start --kubernetes-version v1.10.0
 ```
 
+Start Minikube with CNI plugin enabled to allow network policies
```

Comment:
Can you move this up, i.e. make readers installing Minikube with CNI enabled by default? Also, better say

```
Start Minikube with CNI plugins enabled and install Calico for network policies to take effect:
```

And only after that add a paragraph to tell that a NetworkPolicy capable CNI is not mandatory for Sensu clusters to work, e.g.

```
For testing, a NetworkPolicy capable CNI plugin is not necessary, the operator will install the policy regardless w/o effect.
```